### ISSUE TITLE
Accessibility of image-dominated FXL publications

### ISSUE THREAD

**Issue by murata2makoto** [mattgarrish/epub-revision#2012]
In Japan, most EPUB publications contain almost no text.  I propose to add some remarks about such image-dominated FXL publications,  Here is my proposal.

Some FXL publications contain lots of images and almost no text. Moreover, these images contain rasterized text.  Most EPUB
publications in Japan are such FXL publications.

Image-dominated FXL publications are used for mangas, picture books, and even scholarly publications containing complicated layouts.  Unfortunately, image-dominated FXL publications have almost no accessibility.

One reason for using image-dominated FXL publications is the difficulty to achieve layout fidelity.  There are at least two reasons. First, font embedding is difficult due to the sheer number of characters in Japan.  Second, the mixture of vertical-writing and horizontal-writing makes layout fidelity particularly difficult. In fact, FXL publications containing lots of text have almost never been tried in Japan.

Another reason is that manga is fundamentally image-dominated.  Even if all words are read aloud, manga is not very enjoyable for blind people.  For this reason, some attempts for accessible manga introduce aural descriptions of scenes. and adopt purely aural representations rather than EPUB.  However, it remains to be seen if it is possible to make EPUB manga accessible for dyslexic people, low-vision people, and so forth.  Representing words as text without rasterization in EPUB publications might be good enough.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2022-03-03-epub#section4) on 2022-03-03

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 4. Accessibility of image-dominated FXL publications (issue epub-specs#2012)

_See github issue [epub-specs#2012](https://github.com/w3c/epub-specs/issues/2012)._




**Shinya Takami (高見真也):** FL a11y issue was raised (see issue).  
… MURATA, what do you think?.  

**Murata Makoto:** I do not understand the practical implications.  

**Shinya Takami (高見真也):** (Japanese).  

**Murata Makoto:** Informational document (Japanese).  
… DAISY (Japanese).  

**Shinya Takami (高見真也):** (Japanese).  

**Murata Makoto:** Yes.  
… Will this have any impact on 3.3?.  

**Wendy Reid:** No.  
… These are not published, and would only be WG notes, neither on REC track.  
… We can add something about image dominated publications (or add more).  
… We have been trying to avoid discussing a single content type, but we can attempt to differentiate between mainly image FL and text based FL.  

**Murata Makoto:** I would like to make this more obvious - Image based FL content is not accessible. Period..  

**Wendy Reid:** We wouldn't really put this into core anyway, it would be part of a11y if we did add it.  
… There are a lot of problems, and we really can't standardize yet.  

**Dave Cramer:** Looks like we have a path forward.  
… AOB.  

**Dan Lazin:** I heard back from MDN today, they have approved our proposals to add publishing related content at MDN.  
… Now we just have to write the docs.  

> *Dan Lazin:* See ["Web-related technologies" on MDN](https://developer.mozilla.org/en-US/docs/Related).

**Dan Lazin:** We have an outline for minimum viable, but not time to actually do it now.  

> *Dan Lazin:* See [first outline](https://github.com/dlazin/mdn-yari/blob/main/kumascript/macros/EPUBRef.ejs).

**Dan Lazin:** but we do have an outline if anyone wants to help.  

**Zheng Xu (徐征):** In the CG we have a taskforce for MDN, can we assign it to you?.  

**Dan Lazin:** Yes, I knew it existed and the TF connected me to the MDN people.  

**Zheng Xu (徐征):** I will add you to the CG issue.  

**Dan Lazin:** I do have the previous drafts.  

> *Zheng Xu (徐征):* here is the CG issue.

_See github issue [publishingcg#10](https://github.com/w3c/publishingcg/issues/10)._




**Dave Cramer:** The MDN format is weird.  

---


</details>

**Comment by murata2makoto** [comment]
@wareid 

Thank you for the pull request.  Although it is an improvement, I do not think that this issue can be closed.  I would like text rasterization to be fully explained.  What is it?  Why is it used?  What is problematic?  What can the industry do?

**Comment by murata2makoto** [comment]
@wareid  The merged commit is a significant improvement.  When examples in some genres are added, I might submit another issue.
